communitydelphi.properties

```diff
@@ -1,15 +1,23 @@
 category=Languages
 description=Code Analyzer for Delphi
 homepageUrl=https://github.com/integrated-application-development/sonar-delphi
-archivedVersions=1.0.0,1.1.0,1.2.0,1.3.0,1.4.0,1.5.0,1.6.0,1.7.0,1.8.0,1.9.0,1.10.0,1.11.0,1.12.0,1.12.1,1.12.2,1.13.0
-publicVersions=1.14.0
+archivedVersions=1.0.0,1.1.0,1.2.0,1.3.0,1.4.0,1.5.0,1.6.0,1.7.0,1.8.0,1.9.0,1.10.0,1.11.0,1.12.0,1.12.1,1.12.2,1.13.0,1.14.0
+publicVersions=1.14.1
 
 defaults.mavenGroupId=au.com.integradev.delphi
 defaults.mavenArtifactId=sonar-delphi-plugin
 
+1.14.1.description=Bug fixes
+1.14.1.sqs=[10.8,LATEST]
+1.14.1.sqcb=[24.12,LATEST]
+1.14.1.sqVersions=[9.9,10.7]
+1.14.1.date=2025-03-05
+1.14.1.changelogUrl=https://github.com/integrated-application-development/sonar-delphi/releases/tag/v1.14.1
+1.14.1.downloadUrl=https://github.com/integrated-application-development/sonar-delphi/releases/download/v1.14.1/sonar-delphi-plugin-1.14.1.jar
+
 1.14.0.description=2 new rules, improvements to MixedNames, parsing fixes, bug fixes
-1.14.0.sqs=[10.8,LATEST]
-1.14.0.sqcb=[24.12,LATEST]
+1.14.0.sqs=[10.8,2025.1.*]
+1.14.0.sqcb=[24.12,25.2.*]
```

Comment:
25.3 has just been released. Would you update for that, please?

Reply:
I've updated for 25.3 now.